pylib/openrazer/_fake_driver/razerblackwidowv3problutooth.cfg

```diff
@@ -0,0 +1,26 @@
+[device]
+dir_name = 0003:1532:025B.0001
+name = Razer BlackWidow V3 Pro(Blutooth)
```

Comment:
Is there a chance the daemon isn't picking up bluetooth mode due to this file (as well as the contents) reference bluetooth as 'blutooth'?